adafruit_is31fl3731.py

```diff
@@ -193,15 +265,25 @@ def fill(self, color=None, blink=None, frame=None):
             for col in range(18):
                 self._register(frame, _BLINK_OFFSET + col, data)
 
-    def _pixel_addr(self, x, y):
+    def pixel_addr(self, x, y): #pylint: disable-msg=no-self-use
```

Comment:
you can use `@staticmethod` here and drop the `self`

Reply:
ok